go/backend/archive/archive.go

```diff
@@ -14,8 +14,12 @@ package archive
 
 import (
 	"github.com/Fantom-foundation/Carmen/go/common"
+	"github.com/Fantom-foundation/Carmen/go/common/witness"
 )
 
+// ErrWitnessProfNotSupported is returned when the archive does not support witness proofs.
+const ErrWitnessProfNotSupported = common.ConstError("witness proof not supported")
```

Comment:
I guess it should be `Proof` nit `Prof` ... although a `Prof` can also be a Witness :)
```suggestion
const ErrWitnessProofNotSupported = common.ConstError("witness proof not supported")
```